src/ILCompiler/repro/Program.cs

```diff
@@ -1,13 +1,32 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
I don't think we want to change this simple hello world. If needed, you should add another one as part of our build/test.